src/states/main_game.rs

```diff
@@ -23,6 +23,8 @@ use std::f32::consts::PI;
 
 pub struct MainGameState {
     dispatcher: Dispatcher<'static, 'static>,
+    debug_dispatcher: Dispatcher<'static, 'static>,
+    show_debug: bool,
```

Comment:
It might make sense to move this into the `World` as a resource. Then systems etc. would also have access to it, if necessary.

Reply:
@marot Sure thing! Just added it as a `DebugConfig` resource and updated the code to use that

Reply:
Thanks!